src/app/shared/components/line-item/line-item-edit-dialog/line-item-edit-dialog.component.html

```diff
@@ -26,7 +26,10 @@
       <!-- input (quantity) -->
       <div class="form-group">
         <div class="row">
-          <label class="col-form-label col" translate>product.quantity.label</label>
+          <ish-product-quantity-label
+            translationKey="product.quantity.label"
+            class="col-term col"
```

Comment:
The class `col-term` should be removed in the attribute "class" to have a better horizontal alignment to the quantity input field.